spec/integration/alerts_spec.rb

```diff
@@ -623,6 +655,17 @@ module Hawkular::Alerts::RSpec
       expect(events.size).to be >= 12
     end
 
+    it 'Should list events for multiple tenants' do
+      client = Hawkular::Alerts::Client.new(ALERTS_BASE, creds, options)
+
+      events = client.events(criteria: { 'thin' => true }, tenants: [:hawkular, :foo])
+
+      expect(events).to_not be_nil
+      expect(events.size).to be >= 24
+      expect(events.count { |e| e.tenantId == 'hawkular' }).to be >= 12
+      expect(events.count { |e| e.tenantId == 'foo' }).to be >= 12
```

Comment:
Can you put something like
```ruby
expect(events.count { |e| e.tenantId == 'hawkular' } + events.count { |e| e.tenantId == 'foo' }).to eq events.size
```
Just to be sure that we don't have alerts from any other tenant.